src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.wasm.cs

```diff
@@ -166,11 +178,6 @@ private void ParseStateString(string stateString)
 			var loaded = parts[2].Equals("true", StringComparison.Ordinal);
 			var paused = parts[3].Equals("true", StringComparison.Ordinal);
 
-			if (paused && !_isUpdating)
-			{
-				_playState = null;
-			}
```

Comment:
Are you sure of this change?

Reply:
This one was tricky, I'm still not sure what the purpose of it was in the first place but erasing the play state was causing problems. I know it would cause the animation not to play if `OnJsonChanged` for reasons such as the Background/Foreground Color property changing.